front/src/common/redux/students/students.actions/index.js

```diff
@@ -0,0 +1,6 @@
+import { studentRegistered, registrationCompleted } from './studentRegistrationAction';
```

Comment:
Hey, it is quite a strange folder name `students.actions`. Please, have folder `students` and inside it, file `students.actions.js`

Reply:
Changed folders and files structure